kani-driver/src/args/common.rs

```diff
@@ -50,8 +50,6 @@ pub trait Verbosity {
     /// Whether we should be verbose.
     /// Note that `debug() == true` must imply `verbose() == true`.
     fn verbose(&self) -> bool;
-    /// Whether we should emit debug messages.
-    fn debug(&self) -> bool;
```

Comment:
Was it necessary to remove this?

Reply:
It wasn't necessary to remove it, it was just unused. I can add it back with `#[allow(unused)]` to make clippy happy.